src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx

```diff
@@ -161,7 +161,7 @@ const FirstSquare = ({ editPage }: { editPage: boolean }) => {
       const reachedMaxValidators = !operatorStore.isOperatorRegistrable(operator.validators_count);
       const disabled = reachedMaxValidators || isDeleted;
       const disableCheckBoxes = operatorStore.selectedEnoughOperators;
-      const isInactive = operator.status.toLowerCase() === 'inactive';
+      const isInactive = operator.status.toLowerCase() === 'inactive' || operator.status.toLowerCase() === 'no validators';
```

Comment:
This condition can be replaced with `operator.is_active < 1` - it includes both inactive and no validators case because inactive is 0 and no validators is -1. 
You can create somewhere Enum with these values and use it across all the webapp